### ISSUE TITLE
use of Composition api

### ISSUE THREAD

**Issue by javad-kazemi** [tuandm/laravue#200]
will you the use of Composition api in future?

**Comment by tuandm** [comment]
@javad-kazemi Yes, no doubt it's an interesting thing that needs to bring into this project in the near future.
